x-pack/plugins/observability_solution/apm/public/components/alerting/utils/fields.tsx

```diff
@@ -203,22 +203,50 @@ export function IsAboveField({
   onChange: (value: number) => void;
   step?: number;
 }) {
+  const [thresholdPopoverOpen, serThresholdPopoverOpen] = useState(false);
+  const [isAboveValue, setIsAboveValue] = useState(String(value));
+  const isInvalid = isNaN(Number(isAboveValue));
```

Comment:
nit: This can be a function so you can pass the value -  something like `const isValidNumber = (value: number): boolean => !isNaN(Number(value))` and reuse it on lines 241 (`isValidNumber(thresholdVal)`) and 246 (`!isValidNumber(isAboveValue)`)

Reply:
Sure